engine/template.go

```diff
@@ -233,3 +269,12 @@ func unmarkdown(s *string) string {
 	withoutLinebreaks := strings.ReplaceAll(withoutMarkdown, "\n", " ")
 	return withoutLinebreaks
 }
+
+func unescapeHTML(s string) htmltemplate.HTML {
+	return htmltemplate.HTML(s) //nolint:gosec
+}
+
+func (t *Templates) i18n(messageID string, lang language.Tag) string {
+	localizer := t.localizers[lang]
+	return localizer.MustLocalize(&i18n.LocalizeConfig{MessageID: messageID})
```

Comment:
Kunnen we hier niet altijd unescapeHTML doen en er geen aparte template functie van maken?

Reply:
Ja prima idee, scheelt weer een functie, en dan hoef je niet overal te bedenken of iets wel of niet unescaped moet worden.

Reply:
Done